### ISSUE TITLE
the expected status code should be 202 instead of 201 in blockstorage/v1/volumes/request.go line 45

### ISSUE THREAD

**Issue by dragon753** [gophercloud/gophercloud#1718]
when executed, the actual response status is 202, but in the code, it's 201.
the v3 version is fine.

**Comment by jtopjian** [comment]
Thank you for reporting this. 

Are you sure 202 is correct? I think we would have received a bug report about this a few years ago if this was the case.

**Comment by dragon753** [comment]
Yes, I am sure. My openstack version is 3.8.1, cinder version is 1.11.0.  
when using the v1 sdk, the volume was created, correct response was extracted into the "error", after I changed the sdk to v3, I got the same response in the return value "volume" , and the error is nil.

**Comment by jtopjian** [comment]
I seat up an environment based on Mitaka and was able to reproduce this problem:

```
--- FAIL: TestVolumesCreateDestroy (1.18s)
    blockstorage.go:54: Attempting to create volume: ACPTTESTdQOwXTKgIkUMe1iK
    volumes_test.go:43: Unable to create volume: Expected HTTP response code [200 201] when accessing [POST http://10.1.2.177/volume/v3/5bfa04f074684d73b714df219457f327/volumes], but got 202 instead
```

Though I'm quite sure this has worked in the past. I'm going to patch this to allow both 201 and 202 - I don't see harm in doing that.

Thank you for reporting this!

**Comment by jtopjian** [comment]
This has been merged in #1720. I'm going to close this issue but please let me know if you have any questions.
